packages/core-sdk/src/types/resources/dispute.ts

```diff
@@ -1,56 +1,56 @@
 import { TxOptions } from "../options";
 
-export type WhitelistDisputeTagsResponse = {
-  txHash: string;
-};
-
-export type WhitelistDisputeTagsRequest = {
-  tag: string;
-  allowed: boolean;
-  txOptions?: TxOptions;
+export type Dispute = {
+  targetIpId: `0x${string}`; // The ipId that is the target of the dispute
+  disputeInitiator: `0x${string}`; // The address of the dispute initiator
+  arbitrationPolicy: `0x${string}`; // The address of the arbitration policy
+  linkToDisputeEvidence: string; // The link of the dispute evidence
+  targetTag: string; // The target tag of the dispute
+  currentTag: string; // The current tag of the dispute
 };
 
-export type WhitelistArbitrationPolicyRequest = {
+export type RaiseDisputeRequest = {
+  targetIpId: `0x${string}`;
   arbitrationPolicy: `0x${string}`;
-  allowed: boolean;
+  linkToDisputeEvidence: string;
+  targetTag: string;
+  calldata?: `0x${string}`;
   txOptions?: TxOptions;
 };
 
-export type WhitelistArbitrationPolicyResponse = {
+export type RaiseDisputeResponse = {
   txHash: string;
 };
 
-export type WhitelistArbitrationRelayerRequest = {
-  arbitrationPolicy: `0x${string}`;
-  arbitrationRelayer: `0x${string}`;
-  allowed: boolean;
+export type SetDisputeJudgementRequest = {
+  disputeId: number;
+  decision: boolean;
+  calldata?: `0x${string}`;
   txOptions?: TxOptions;
 };
 
-export type WhitelistArbitrationRelayerResponse = {
+export type SetDisputeJudgementResponse = {
   txHash: string;
+  disputeId?: bigint;
+  decision?: boolean;
+  data?: `0x${string}`;
 };
 
-export type RaiseDisputeRequest = {
-  targetIpId: `0x${string}`;
-  arbitrationPolicy: `0x${string}`;
-  linkToDisputeEvidence: string;
-  targetTag: string;
+export type CancelDisputeRequest = {
+  disputeId: number | string | bigint;
```

Comment:
I think we just use either number or string here. Otherwise users have to figure out which types to use.

Reply:
I think bigint is ok here. If a user uses ethers/wagmi to read directly from the contract, that output is bigint type.